src/Components/Steps/HouseholdMembers/PersonIncomeBlock.css

```
@@ -45,11 +45,11 @@
 }
 
 .income-block-container {
-  padding: 0 0 3rem 0;
-  margin-bottom: 1rem;
+  padding: 0 0 1rem 0;
```

Stylelint (17.2.0)

[error] Expected "0 0 1rem 0" to be "0 0 1rem" (shorthand-property-no-redundant-values)

(shorthand-property-no-redundant-values)